pallets/protos/rpc/runtime-api/src/lib.rs

```diff
@@ -4,8 +4,21 @@ use sp_chainblocks::Hash256;
 use sp_std::vec::Vec;
 
 sp_api::decl_runtime_apis! {
-		pub trait ProtosApi<Tags> where
-				Tags: Codec, {
-				fn get_by_tag(tags: Tags) -> Option<Vec<Hash256>>;
-		}
+	pub trait ProtosApi<Tags, AccountId>
+	where
+		Tags: Codec,
+		AccountId: Codec
+	{
+		fn get_by_tag(tags: Tags) -> Option<Vec<Hash256>>;
+
+		fn get_by_tags(tags: Vec<Tags>, owner: Option<AccountId>, limit: u32) -> Vec<Hash256>;
```

Comment:
wrong signature, where is `from` and `desc`?

Reply:
also `at`?

Reply:
`from` and `desc` are parameters in the actual Custom RPC function (`pallets/protos/rpc/src/lib.rs`) but not in the Runtime API (because right now I'm not sure what `from` or `desc` does - could you clarify what they are supposed to? @sinkingsugar )

Reply:
`at` is a parameter in in the actual Custom RPC function (`pallets/protos/rpc/src/lib.rs`)

Reply:
uh? we discussed...
`from` is the index to START from
and `desc` descending (vs ascending)

I see that this is not implemented, TODO